DeepSpeech.py

```diff
@@ -446,6 +466,22 @@ def train():
         dev_sets = [create_dataset([csv], batch_size=FLAGS.dev_batch_size, train_phase=False) for csv in dev_csvs]
         dev_init_ops = [iterator.make_initializer(dev_set) for dev_set in dev_sets]
 
+    # The transfer learning approach here need us to supply the layers which we
+    # want to exclude from the source model.
+    # Say we want to exclude all layers except for the first one, we can use this:
+    #
+    #    drop_source_layers=['2', '3', 'lstm', '5', '6']
+    #
+    # If we want to use all layers from the source model except the last one, we use this:
+    #
+    #    drop_source_layers=['6']
+    #
+
+    if FLAGS.load == "transfer":
+        drop_source_layers = ['2', '3', 'lstm', '5', '6'][-int(FLAGS.drop_source_layers):]
```

Comment:
nit: shouldn't need `int(...)` here since the flag is already defined as an integer.